db/schema.rb

```diff
@@ -10,7 +10,8 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_03_31_085103) do
+ActiveRecord::Schema.define(version: 2020_03_30_130958) do
```

Comment:
This looks like it's going earlier rather than later. Is that correct?

Reply:
That'll be my mashed brain doing a bad rebase 😆  This PR has not been kind to me